docs/_dapp/native.md

```diff
@@ -3,64 +3,89 @@ title: Go API
+about Go project layouts, import paths or any other standard methodologies. If you are new to Go, 
+consider reading [Getting Started with Go][go-guide] first.*
 
-Our reusable Go libraries focus on four main usage areas:
+## Overview
+
+Geth's reusable Go libraries focus on three main usage areas:
 
 - Simplified client side account management
 - Remote node interfacing via different transports
 - Contract interactions through auto-generated bindings
-- In-process Ethereum, Whisper and Swarm peer-to-peer node
 
-You can watch a quick overview about these in Peter's (@karalabe) talk titled "Import
-Geth: Ethereum from Go and beyond", presented at the Ethereum Devcon2 developer conference
-in September, 2016 (Shanghai). Slides are [available here][peter-slides].
+The libraries are updated synchronously with the Geth Github repository. 
+The Go libraries can be viewed in full at [Go Packages][go-pkg-link]
+
+The benefits of using the Go libraries include easy client-side [account management](/docs/dapp/native-accounts)
+that allows users to keep their private keys safely encrypted locally and make their own
+security decisions. The Go libraries allow dapp developers to spend more of their time
+building their dapps rather than implementing complex protocols such as encoding and
+decoding function calls to smart contracts - this can all be abstracted away using
+Geth's Go libraries.
+
+Péter Szilágyi (@karalabe) gave a high level overview of the Go libraries in 
+a talk at DevCon2 in Shanghai in 2016. The slides are still a useful resource
+([available here][peter-slides]) and the talk itself can be viewed by clicking
+the image below (it is also archived on [IPFS][ipfs-link]).
 
-[![Peter's Devcon2 talk](https://img.youtube.com/vi/R0Ia1U9Gxjg/0.jpg)](https://www.youtube.com/watch?v=R0Ia1U9Gxjg)
+[![Peter's Devcon2 talk](/static/images/devcon2_labelled.webp)](https://www.youtube.com/watch?v=R0Ia1U9Gxjg)
 
 ## Go packages
 
-The `go-ethereum` library is distributed as a collection of standard Go packages straight
-from our GitHub repository. The packages can be used directly via the official Go toolkit,
-without needing any third party tools. External dependencies are vendored locally into
-`vendor`, ensuring both self-containment as well as code stability. If you reuse
-`go-ethereum` in your own project, please follow these best practices and vendor it
-yourself too to avoid any accidental API breakages!
+The Geth library is distributed as a collection of standard Go packages straight from Geth's GitHub
+repository. The packages can be used directly via the official Go toolkit, without needing any 
+third party tools. External dependencies are vendored locally into `vendor`, ensuring both 
+self-containment and code stability. When Geth is used in downstream projects these best
+practices (packing dependencies into a local `vendor`) should be followed there too to 
+avoid any accidental API breakages.
```

Comment:
```suggestion
third party tools.
```
The 'vendor' was something that predated go modules, which now is fully supported and makes a lot of the problems around external dependencies go away. Or, at least become a lot more standardized.